src/components/calcite-action/calcite-action.e2e.ts

```diff
@@ -100,8 +100,8 @@ describe("calcite-action", () => {
   });
 
   it("should be accessible", async () => {
-    accessible(`<calcite-action text="hello world" label="hi"></calcite-action>`);
+    await accessible(`<calcite-action text="hello world" label="hi"></calcite-action>`);
```

Comment:
would it be clearer for all the tests to use the `async await` pattern rather than the `(param1, param2, …, paramN) => expression` pattern?

Reply:
Maybe. I don't want to enforce one vs the other though. I like to keep it concise and expressive whenever possible, especially since IDEs make it easy.